main/webapp/modules/core/scripts/index/default-importing-controller/controller.js

```diff
@@ -270,7 +270,9 @@ Refine.DefaultImportingController.prototype.getPreviewData = function(callback,
 			  callback(result);
 		  },
         "jsonp"
-      );
+	   ).fail(function(jqXhr, textStatus, errorThrown) {
+		  alert($.i18n('core-index/rows-loading-failed'));
+	  });
```

Comment:
```suggestion
	   ).fail(() => { alert($.i18n('core-index/rows-loading-failed')); });
```
Since you don't use the param, might as well use an arrow function, don't you think? It makes the code simplier.

Regards, Antoine